### DISCUSSION TITLE
Installing an extension that's being develop

### DISCUSSION THREAD

**Discussion by fpmirabile** [gitpod-io/openvscode-server#471] [Q&A]
So, for a CI/CD I would love to install remotely through the CI/CD the extension I'm developing and use this to test e2e through playwright.
I can't find a way of installing it, remotely and throguh docker.
Is it possible?

**Comment by jeanp413** [comment]
@fpmirabile there's an official module that you can use for testing https://github.com/microsoft/vscode-test, does that work for your case?
